src/main/java/org/terasology/herbalism/system/HerbDropAuthoritySystem.java

```diff
@@ -166,12 +153,13 @@ public void onGeneratedHerbDestroyed(DoDestroyEvent event, EntityRef entity, Gen
         if (random.nextFloat() < chanceOfBlockDrop) {
             // Get the base genome and world position of the old herb.
             final String herbBaseGenome = herbComp.herbBaseGenome;
-            final Vector3f position = locationComp.getWorldPosition();
+            final Vector3f position = locationComp.getWorldPosition(new Vector3f());
```

Comment:
@pollend you said on discord that the usage of this position is basically rounding "half up". So, should we just replace this with
```suggestion
            final Vector3i position = new Vector3i(locationComp.getWorldPosition(new Vector3f()), RoundingMode.HALF_UP);
```